<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlossaryExtractorDialog</class>
 <widget class="QDialog" name="GlossaryExtractorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Minimalna liczba słów we frazie:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QSpinBox" name="_minWordCount"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Maksymalna liczba słów we frazie:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QSpinBox" name="_maxWordCount"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Minimalna liczba powtórzeń frazy:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QSpinBox" name="_minOccurence"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Maksymalna liczba powtórzeń frazy:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QSpinBox" name="_maxOccurence"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Znaleziono fraz:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QLineEdit" name="_foundCount"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="_close">
     <property name="text">
      <string>Zakończ</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="_find">
     <property name="text">
      <string>Wyszukaj</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QPushButton" name="_save">
     <property name="text">
      <string>Zapisz</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Zero oznacza brak limitu.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
